ceph/ceph_admin/shell.py

```diff
@@ -1,11 +1,13 @@
 """Interface to cephadm shell CLI."""
 import logging
-from typing import List
+from copy import deepcopy
+from typing import Dict, List
 
+from .common import config_dict_to_string
 from .typing_ import CephAdmProtocol
 
 LOG = logging.getLogger()
-BASE_CMD = ["cephadm", "-v", "shell", "--"]
+BASE_CMD = ["cephadm", "-v", "shell"]
```

Comment:
Thanks for extending our new functionality here... this keeps it consistent with other objects.